src/platform/i386/boot_comp.c

```diff
@@ -72,8 +74,8 @@ boot_pgtbl_mappings_add(struct captbl *ct, capid_t pgdcap, capid_t ptecap, const
 		paddr_t pf    = chal_va2pa(p);
 		u32_t   mapat = (u32_t)user_vaddr + i * PAGE_SIZE, flags = 0;
 
-		if (uvm && pgtbl_mapping_add(pgtbl, mapat, pf, PGTBL_USER_DEF)) assert(0);
-		if (!uvm && pgtbl_cosframe_add(pgtbl, mapat, pf, PGTBL_COSFRAME)) assert(0);
+		if (uvm && pgtbl_mapping_add(pgtbl, mapat, pf, X86_USER_DEF)) assert(0);
```

Comment:
Should still be namespaced to page-tables, right?

`X86_PGTBL_USER_DEF`, for example.

Reply:
Don't think that this change has been made yet...

Reply:
Now namespaced